src/main/java/com/CezaryZal/controller/LoginController.java

```diff
@@ -19,8 +23,8 @@ public LoginController(LoginService loginService) {
     }
 
     @PostMapping("/login")
-    public ResponseEntity<String> getTokenByUserLogin(@RequestBody UserLogin inputUserLogin){
-        return loginService.getTokenByUserLogin(inputUserLogin);
+    public ResponseEntity<String> getTokenByUserLogin(@RequestBody UserLogin inputUserLogin) throws NameNotFoundException, AccountNotFoundException {
```

Comment:
czym się w sensie logicznym różni NameNotFoundException od AccountNotFoundException? Czy to są dwa odrębne wyjątki?

Reply:
Exception NameNotFoundException został z poprzedniej logiki, nieusunołem rzucanego wyjątku.
Usunięte.
nazwę metode zmieniłem zgodnie z jej funkcją: `throwExceptionIfUserIsNotActive`